src/main/java/org/galatea/starter/service/IexClient.java

```diff
@@ -32,4 +35,13 @@ public interface IexClient {
   @GetMapping("/tops/last?token=${spring.rest.iexToken}")
   List<IexLastTradedPrice> getLastTradedPriceForSymbols(@RequestParam("symbols") String[] symbols);
 
+  /**
+   * Get historical pricing data for the given symbol on the given date. See https://iexcloud.io/docs/api/#historical-prices
+   *
+   * @param symbol the symbol to retrieve data about
+   * @param date the date which should be queried
+   * @return historical pricing data for the given symbol on the given date
+   */
+  @GetMapping("/stock/{symbol}/chart/date/{date}?chartByDay=true&token=${spring.rest.iexToken}")
```

Comment:
The IEX API docs show multiple possible values for range, rather than just `/stock/{symbol}/chart/date/{date}`. For example, `/stock/twtr/chart/max`, `/stock/twtr/chart/1m`. Try supporting those as well.